### ISSUE TITLE
Default subproject for implicit completion

### ISSUE THREAD

**Issue by paour** [gradle/gradle-completion#46]
The `GRADLE_COMPLETION_UNQUALIFIED_TASKS` option causes the completion script to go over all subprojects, potentially pulling in too many tasks. Android apps generally always use `:app:` for the main subproject, could another env var direct the script to autocomplete for that by default?

**Comment by eriwen** [comment]
Yes, I'd be very happy to consider a PR here especially that makes completion better for Android projects. Is that something you'd be willing to submit?

**Comment by paour** [comment]
With my bash skills, you'd have to rewrite whatever I contributed ;-).

What I had in mind was not necessarily Android-specific: the variable could be set to whatever the "usual" default subproject the developer uses.

**Comment by eriwen** [comment]
Okay, thanks for the clarification. That is helpful.

**Comment by mwos-sl** [comment]
I think that could be easily handled by: https://github.com/gradle/gradle-completion/issues/88
I've already implemented such task. It can be easily extended. I can share the code.
To apply it, this PR needs to be accepted: https://github.com/gradle/gradle-completion/pull/89

**Comment by eskatos** [comment]
This is the intended behavior, and there is no plan to change it. Therefore, this issue will be closed.

***

We favor this instead:
* https://github.com/gradle/gradle-completion/issues/36
